### DISCUSSION TITLE
Restarting rust-analyzer without restarting nvim?

### DISCUSSION THREAD

**Discussion by damccull** [mrcjkb/rustaceanvim#485] [Q&A]
With lspconfig, I used to be able to call :LspRestart to restart the rust-analyzer lsp. That doesn't _seem_ to do anything with rustaceanvim. Am I just misunderstanding and it's actually restarting it, or is there some other way to restart rust-analyzer with the rustaceanvim plugin? Thanks!

**Comment by mrcjkb** [comment]
`:h rustaceanvim`. Commands are right at the top :smile: 

rustacranvim doesn't use lspconfig, so you can't use lspconfig commands.

**Comment by damccull** [comment]
Haha. Dang, I'm so sorry. I am so used to open source projects having bad or non-existing docs...I will try to remember to check the helps for rustaceanvim first :)

I appreciate the reply, though! This is one of the best nvim plugins I've ever seen.
